.env

```diff
@@ -13,7 +13,7 @@ MINER_PRIVATE_KEY=da81490c7efd5a95398a3846fa57fd17339bdf1b941d102f2d3217ad29785f
 DISABLE_POW_VALIDATION=1
 
 # Replica counts for different node types
-COMMON_NODE_REPLICAS=2
-MINING_NODE_REPLICAS=1
+COMMON_NODE_REPLICAS=1
+MINING_NODE_REPLICAS=0
```

Comment:
Should the default environment have no mining nodes?

Reply:
The mining node does CPU mining that ends up using a lot of resources, I think it should default to 0, the use case for testing mining like that is very rare afaik.

Reply:
That sound reasonable.